packages/dialtone-css/lib/build/less/components/datepicker.less

```diff
@@ -14,14 +14,15 @@
   --datepicker-width: calc(var(--dt-size-300) * 75); // A bit of a magic number as default fixed size, may use CSS utilities to override
   --datepicker-button-size: calc(var(--dt-size-600) - var(--dt-size-200)); // Emulates SM button since Vue component in lieu of Vue component using DT button
 
+  box-sizing: border-box;
   width: var(--datepicker-width);
   padding: var(--dt-space-500);
-  box-sizing: border-box;
 
   &__hd {
     padding: 0 var(--dt-space-300);
   }
 
+  /* stylelint-disable-next-line block-no-empty */
   &__bd {
   }
```

Comment:
Why not removing this?

Reply:
It looks like it was put there to define the structure so future properties could be put there. I'm fine either way, could remove I suppose

Reply:
removed